### ISSUE TITLE
Конфликт бандлов Source и Zen Coding: Cmd + /

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/zen-coding#110]
```
Если включен обязательный по умолчанию 
бандл Source, то нажатие Cmd + / приводит к 
конфликту горячих клавиш и такому меню — 
см. аттач.

Решение: убрать макрос Toggle Comment из бандла 
или переключить его на Cmd + \ (как в 
Espresso).

Он ешё и страшно бажит: если вызвать его 
внутри style в голове документа, то css-коммент 
появится в самом начале документа — только 
в TextMate. Но насчёт этого отдельный баг.
```

---

Original issue reported on code.google.com by `pepelsbey` on 19 Feb 2010 at 2:50

Attachments:
- [comment.png](https://storage.googleapis.com/google-code-attachments/zen-coding/issue-110/comment-0/comment.png)

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `serge....@gmail.com` on 1 Feb 2011 at 10:56
- Added labels: **Milestone-0.7**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v0.7
```

---

Original comment by `serge....@gmail.com` on 2 Feb 2011 at 8:15
- Changed state: **Fixed**
